src/Containers/FeatureDisplay/FeatureDisplay.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import SmallText from '@Text/SmallText';
+import styled from 'styled-components';
+import { flex, media } from '@Utils/Mixins';
+import { Check } from '@styled-icons/boxicons-regular/Check';
+import { ShowChart } from '@styled-icons/material/ShowChart';
+import { Card } from '../Card/Card';
+
+export interface IFeatureDisplayProps {
+    featureTitle: string;
+    featureSubTitle: string;
+    featureImage: string;
+    imageTitle: string;
+    imageTags: string[];
+    imageTagColors: string[];
+    highlightTexts: string[];
+    featureFooter: string;
+}
+
+export const FeatureDisplay: React.FC<IFeatureDisplayProps> = ({
+    featureTitle,
+    featureSubTitle,
+    featureImage,
+    imageTitle,
+    imageTags,
+    imageTagColors,
+    highlightTexts,
+    featureFooter
+}) => {
+    const getHighlightText = () => {
+        return highlightTexts.map(
+            (highlightText: string) => (
+                <SmallText size="h7" key={highlightText}>
+                    <Tick />
+                    {' '}
+                    {highlightText}
+                    <div />
+                </SmallText>
+            )
+        );
+    }
+    const getImageTags = () => {
```

Comment:
Remember to write JSDOC to give these functions a clear description of what they do :)